internal/service/ldapconfiguration/resource_ldap_configuration.go

```diff
@@ -242,7 +242,13 @@ func resourceUpdate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 
 func resourceDelete(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
 	connV2 := meta.(*config.MongoDBClient).AtlasV2
-	_, _, err := connV2.LDAPConfigurationApi.DeleteLDAPConfiguration(ctx, d.Id()).Execute()
```

Comment:
is there any value in preserving this delete call, or it is not related?

Reply:
I don't think there is value, the API name is misleading

Reply:
@AgustinBettati yes looking at the failing test, it's because the (no) dependency between the resource and ds